README.md

```diff
@@ -91,6 +91,16 @@ If you have installed Clang as an external Xcode toolchain (using the `install-x
 make run-tests
 ```
 
+### Android
+
+It requires a root device or an emulator:
+
+```
+make
+adb push tests/tests /data
+adb shell /data/tests
```

Comment:
I only tried with an emulator running API 22. The goal of running the unit tests is to ensure that we don't introduce stuff which isn't compatible with Android.